srcpkgs/epson-inkjet-printer-escpr2/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'epson-inkjet-printer-escpr2'
 pkgname=epson-inkjet-printer-escpr2
-version=1.1.24
+version=1.1.53
 revision=1
-archs="x86_64 i686 aarch64 armv7l"
+archs="x86_64* i686 aarch64 armv7l"
```

Comment:
for musl, it only works on x86_64?

Reply:
Cross compilation for aarch64-musl and armv7l-musl didn't work.
Should i test i686-musl? (if so, natively on x86_64-musl or cross?)

Reply:
No, don't change this line.
They're linked with pre-built `libescpr2.a`, which is shipped with the distfiles. The pre-built libraries was built with glibc.